drafts/25-xxx-specifications.txt

```diff
@@ -567,11 +567,121 @@ nd20. The result of evaluating a processor-dependent directive is processor-depe
 5 Expressions allowed in #if and #elif directives
 =================================================
 
-5.1 The 'defined' operator
---------------------------
-
-6 Expression evaluation in #if and #elif directives
-===================================================
+ex05. Controlling expressions are made up of "primary expressions" and
+      operators applied to subexpressions.
+
+ex10. When evaluating a #if or #elif directive, preprocessing
+      evaluates and expands all object-like macros and function-like
+      macro calls to create a token-list of the expression to be
+      evaluated.
+
+ex15. The resulting list of tokens shall be a valid expression
+      comprised of primary expressions and operators as described
+      below.
+
+ex15. Preprocessing computes the integer value of this expressions
+      using the greatest integer range available to the processor.to
+      determine the truth or falsity of the controlling expression.
+
+ex20. The evaluation of the expression shall not generate a
+      computational error (such as divide by zero).
```

Comment:
Maybe we should be more explicit about what happens if a computational error occurs instead?
```suggestion
ex20. The processor shall reject a program if evaluation of
      the expression generates a computational error (such 
      as divide by zero).
```